bamboo/README.md

```diff
@@ -1,10 +1,10 @@
-# Plan - Nightly Develop
+# Nightly Testing of Nightly Develop
```

Comment:
I think the title should maybe be something more along the lines of "LBANN CI" or something more general. Then maybe below we state that every test in this directory runs in our nightly plan